recipes/feht/meta.yaml

```diff
@@ -12,7 +13,8 @@ source:
   md5: '{{ hash_value }}'
 
 build:
-  number: 0
+  number: 1
```

Comment:
Please set build number to 0 for new versions.

Reply:
Ah, thanks for catching that!